### PR TITLE
Create WhatIsInclusivity.md

### LINKED ISSUE #1935: Empty what-is articles.
We need what-is articles for the following:

- [ ] * [Programming Languages](https://bssw.io/items?topic=programming-languages)
- [ ] * [High-Performance Computing (HPC)](https://bssw.io/items?topic=high-performance-computing-hpc)
- [ ] * [Cloud Computing](https://bssw.io/items?topic=cloud-computing)
- [ ] * [Big Data](https://bssw.io/items?topic=big-data)
- [x] * [Inclusivity](https://bssw.io/items?topic=inclusivity) (by @rinkug)
- [ ] * [Personal Productivity and Sustainability](https://bssw.io/items?topic=personal-productivity-and-sustainability)(@pagrubel) PR #1944

### PR DESCRIPTION
**Pull request by rinkug**
# Description

Author: @rinkug 
EB Member: @markcmiller86 
Partly Resolves #1935 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [ ] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [ ] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [ ] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [ ] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [ ] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

### CODE REVIEW

**Comment by rinkug** [comment]
@markcmiller86 : can you give this a quick read and modify what you want. If modifications are significant, then pls add yourself as an author as well

**Comment by markcmiller86** [comment]
I'll take a closer look no later than Monday.

**Comment by markcmiller86** [comment]
@rinkug do you by any chance have any source material you could share that you used for this?

File: Articles/WhatIs/WhatIsInclusivity.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+People inclusivity extends beyond code and team dynamics, emphasizing the need for a scientific computing community that is welcoming to individuals from all walks of life. Breaking down barriers related to gender, race, ethnicity, and other factors is essential for creating an environment where everyone feels valued and able to contribute. Initiatives that prioritize mentorship, education, and outreach can play a crucial role in fostering inclusivity in the broader scientific computing community. By actively addressing and dismantling systemic barriers, the scientific computing field can become a more equitable and enriching space for all its participants.
```
Comment (markcmiller86) [inline Articles/WhatIs/WhatIsInclusivity.md:16]: Not sure how *people* inclusivity is different from *teams* inclusivity.

**Review by markcmiller86 — COMMENTED** [review]
@rinkug thanks for starting this. I am making a few comments here now while I consider proposing more detailed changes.

**Comment by rinkug** [comment]
#1943 is a better what-is article. Closing this in favor of that.

**Pull request closed without merging**